aws-batch/cancel.aws.batch.sh

```diff
@@ -0,0 +1,20 @@
+export $(grep -v '^#' .env | xargs)
+echo $AWS_DEFAULT_REGION
+for i in $(aws batch list-jobs --job-queue head_queue --job-status running --output text --query "jobSummaryList[*].[jobId]")
```

Comment:
Yes, since if we don't cancel the head node first, the node will keep submitting new jobs. So that's why we need to terminate the head node first, cancel the upcoming worker node job, and then terminate the current worker node runs.

Reply:
Maybe you can add your explanation as a comment in the code?